cinema/views.py

```diff
@@ -44,14 +46,15 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
     def get_serializer_class(self):
         if self.action == "list":
             return MovieSessionListSerializer
+
         if self.action == "retrieve":
             return MovieSessionDetailSerializer
+
         return self.serializer_class
 
     def get_queryset(self):
         queryset = self.queryset
-        if self.action in ("list", "retrieve"):
-            queryset = queryset.prefetch_related("movie", "cinema_hall")
+        queryset = queryset.prefetch_related("movie", "cinema_hall")
```

Comment:
what is the idea of overriding get_queryset if you can build this query in queryset attribute directly